cmd/antrea-agent/agent.go

```diff
@@ -309,14 +310,19 @@ func run(o *Options) error {
 	if err != nil {
 		return fmt.Errorf("error generating Cipher Suite list: %v", err)
 	}
+	ipFamily := ip.V4Family
+	if nodeConfig.NodeIPAddr.IP.To4() == nil {
+		ipFamily = ip.V6Family
+	}
```

Comment:
another place we will need to update when we support dual Node addresses :)

Reply:
Yes. I added a TODO comment.